x-pack/plugins/security_solution/public/detection_engine/rule_response_actions/response_actions_form.tsx

```diff
@@ -92,7 +87,7 @@ export const ResponseActionsForm = ({
       <ResponseActionsHeader />
       {uiFieldErrors?.length ? (
         <>
-          <p css={FieldErrorsContainer}>
+          <p>
```

Comment:
Should this be a `div` instead?

Reply:
I kept it as <p> since it was that before, but I think it doesn't really matter for us. Even the styles I removed didn't seem to make any impact 🤯